app/main.py

```diff
@@ -1,2 +1,96 @@
+from typing import Any, Hashable
+
+
+class Node:
+    def __init__(self, key: Hashable, hash_key: int, value: Any) -> None:
+        self.key = key
+        self.hash_key = hash_key
+        self.value = value
+
+
 class Dictionary:
-    pass
+    def __init__(
+            self,
+            initial_capacity: int = 8,
```

Comment:
init should take these params - they are part of internal implementation and shouldnt be set from outside

Reply:
why do you think it is necessary to pass capacity on init?